src/anm/animation/animation.js

```diff
@@ -721,6 +727,7 @@ var FONT_LOAD_TIMEOUT = 10000; //in ms
  * @method loadFonts
  * @private
  */
+var URL2FG = 'http://fonts.googleapis.com/css?family=';
```

Comment:
Probably that one should be `URL2GF`? But I'd prefer some less abbreviated name anyway :)